config/jobs/etcd-druid/etcd-druid-build-dev-images.yaml

```diff
@@ -0,0 +1,49 @@
+postsubmits:
+  gardener/etcd-druid:
+    - name: post-etcd-druid-build--dev-images
+      cluster: gardener-prow-trusted
+      run_if_changed: '^go\.mod|^hack\/tools\/image|^hack\/tools\.mk'
+      branches:
+        - ^master$
+      annotations:
+        description: Build etcd-druid development images on master branch
+        testgrid-dashboards: gardener-etcd-druid
+        testgrid-days-of-results: "60"
+      decorate: true
+      max_concurrency: 1
+      reporter_config:
+        slack:
+          channel: prow-alerts
+      spec:
+        serviceAccountName: image-builder
+        containers:
+          - name: image-builder
+            image: europe-docker.pkg.dev/gardener-project/releases/ci-infra/image-builder:v20241111-d15d824
+            command:
+              - /image-builder
+            args:
+              - --log-level=info
+              - --docker-config-secret=gardener-prow-gcr-docker-config
+              - --registry=europe-docker.pkg.dev/gardener-project/releases/ci-infra
+              - --target=etcd-druid
+              - --context=hack/tools/image
```

Comment:
I'm not sure what you are trying to build here 😅

Reply:
I found a concourse job which builds dev images on HEAD updates. Thus, this whole jobs might not be needed, does it?

Reply:
Thanks for providing the info over chat. I have made the corrections for etcd-druid and also for dwd